packages/odata-common/src/request-builder/request-builder-base.ts

```diff
@@ -53,6 +53,7 @@ export abstract class MethodRequestBuilder<
    * Set timeout for requests towards the target system given in the destination.
    * @param timeout - Value is in milliseconds and default value is 10000 (10 seconds).
    * @returns The request builder itself, to facilitate method chaining.
+   * @deprecated Options is replaced by the more general {@link https://sap.github.io/cloud-sdk/docs/js/guides/resilience middleware approach} in version 3.0 of the SDK.
```

Comment:
```suggestion
   * @deprecated The timeout configuration is replaced by the more general {@link https://sap.github.io/cloud-sdk/docs/js/guides/resilience middleware approach} in version 3.0 of the SAP Cloud SDK.
```

[pp] Strictly speaking this is not an option, but a function, that's why I'd suggest using `configuration`.